service-rpc/src/main/java/org/apache/iotdb/rpc/TSStatusCode.java

```diff
@@ -63,10 +63,19 @@
   READ_ONLY_SYSTEM_ERROR(502),
   DISK_SPACE_INSUFFICIENT_ERROR(503),
   START_UP_ERROR(504),
+
   WRONG_LOGIN_PASSWORD_ERROR(600),
   NOT_LOGIN_ERROR(601),
   NO_PERMISSION_ERROR(602),
   UNINITIALIZED_AUTH_ERROR(603),
+
+  // TODO-Cluster: update docs when ready to merge
+  PARTITION_NOT_READY(700),
+  TIME_OUT(701),
```

Comment:
what for? maybe we need to separate the time_out between the single node and cluster.

Reply:
What "what for"? These are error codes for the distributed version, and certainly, you will not have a node's communication time out with another node in the stand-alone version because there is only one node.